Sources/Swift/Integrations/SessionReplay/Preview/SentryMaskingPreviewView.swift

```diff
@@ -0,0 +1,67 @@
+#if (os(iOS) || os(tvOS)) && !SENTRY_NO_UIKIT
+import Foundation
+import UIKit
+
+@objcMembers
+class SentryMaskingPreviewView: UIView {
+    private class PreviewRederer: ViewRenderer {
+        func render(view: UIView) -> UIImage {
+            return UIGraphicsImageRenderer(size: view.frame.size, format: .init(for: .init(displayScale: 1))).image { _ in
+                // Creates a transparent image of the view size that will be used to drawn the redact regions.
+                // Transparent background is the default, so no additional drawing is required.
+                // Left blank on purpose
+            }
+        }
+    }
+    
+    private let photographer: SentryViewPhotographer
+    private var displayLink: CADisplayLink?
+    private var imageView = UIImageView()
+    
+    var opacity: Float {
+        get { return Float(imageView.alpha) }
+        set { imageView.alpha = CGFloat(newValue)}
+    }
+    
+    init(redactOptions: SentryRedactOptions) {
+        self.photographer = SentryViewPhotographer(renderer: PreviewRederer(), redactOptions: redactOptions)
+        super.init(frame: CGRect(x: 0, y: 0, width: 100, height: 100))
```

Comment:
Could we use `CGRect.zero` here, since we'll set the frame to the window bounds in `-[SentrySessionReplayIntegration showMaskPreview:]`, and to the superview bounds in `didMoveToSuperview`?

Reply:
Yes. I will change. Thanks!!